modules/cql/src/blaze/elm/ratio.clj

```diff
@@ -36,6 +37,22 @@
      (p/divide (:denominator x) (:numerator x))))
 
 
+;; 22.29. ToRatio
+(extend-protocol p/ToRatio
+  String
+  (to-ratio [s]
+    ;; (+|-)?#0(.0#)?('<unit>')?:(+|-)?#0(.0#)?('<unit>')?
+    (when
```

Comment:
You can remove the regex check because it's done at the quantity level. See also adding negative tests.

Reply:
The regex check is still there. Was this an oversight or intensional?